experimenter/tests/integration/nimbus/pages/experimenter/branches.py

```diff
@@ -153,6 +164,23 @@ def add_screenshot_buttons(self):
             "branch add screenshot button",
         )
 
+    def _get_feature_value(self, editor_element):
+        return "\n".join(
+            line.text
+            for line in editor_element.find_elements(By.CSS_SELECTOR, ".cm-line")
+        )
+
+    def _set_feature_value(self, editor_element, text):
+        line = editor_element.find_elements(By.CSS_SELECTOR, ".cm-line")[-1]
+        line.click()
+
+        actions = ActionChains(self.driver)
+        actions.send_keys(text)
+        actions.perform()
```

Comment:
Do you need to use ActionChains here? Will sending the keys to the element not work without an action?

Reply:
I couldn't call send_keys on the element because it was a regular div, not an input element.